main_logic/omni_realtime_client.py

```
@@ -1104,81 +1104,108 @@ async def stream_image(self, image_b64: str) -> None:
+            logger.info("prime_context: skipping empty content")
+            return
+
+        if self._is_gemini:
+            # Gemini Live API 没有 session.update 机制，只能通过
+            # send_client_content 注入上下文（会创建 user turn）。
+            # on_response_done 由 _handle_messages_gemini 自然触发。
+            await self._create_response_gemini(text)
+            return
+
+        if not skipped and "qwen" not in self._model_lower:
+            # skipped=False：需要模型主动响应（任务结果汇报）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)